addons/mail/static/src/messaging/entity/chat_window_manager/chat_window_manager.js

```diff
@@ -320,35 +355,89 @@ function ChatWindowManagerFactory({ Entity }) {
             return visual;
         }
 
-        //----------------------------------------------------------------------
-        // Private
-        //----------------------------------------------------------------------
-
-        /**
-         * @override
-         */
-        _update(data) {
-            const {
-                /**
-                 * List of ordered chat windows (list of local ids)
-                 */
-                _ordered = this._ordered || [],
-                isHiddenMenuOpen = this.isHiddenMenuOpen || false,
-            } = data;
-
-            Object.assign(this, {
-                _ordered,
-                isHiddenMenuOpen,
-            });
-        }
-
     }
 
+    ChatWindowManager.entityName = 'ChatWindowManager';
+
     ChatWindowManager.fields = {
+        /**
+         * List of ordered chat windows (list of local ids)
```

Comment:
I would call it _orderedChatWindows then, `_ordered` is really not obvious what it is. (same for other similar fields)